apollo-router/src/services/mod.rs

```diff
@@ -284,8 +284,8 @@ assert_impl_all!(QueryPlannerRequest: Send);
 /// [`Context`] and [`QueryPlanOptions`] for the request.
 #[derive(Clone, Debug)]
 pub struct QueryPlannerRequest {
-    /// Original request to the Router.
-    pub originating_request: http_ext::Request<Request>,
+    pub query: String,
+    pub operation_name: Option<String>,
```

Comment:
Should we put all the `graphql::Request` instead ?

Reply:
BTW it's not a request change, my mistake

Reply:
I don't think we should put the entire request, because variables should not be sent to the query planner. The assumption here is that everything in the query planner request will be used as a key for the cache